test/format/format.sh

```diff
@@ -362,6 +367,11 @@ format()
 		args=${smoke_list[$smoke_next]}
 		smoke_next=$(($smoke_next + 1))
 		echo "$name: starting smoke-test job in $dir ($(date))"
+	elif [[ $timing_stress_split_test -ne 0 ]]; then
+		args=$format_args
+		for k in {1..7}; do
+			args+=" timing_stress_split_$k=$(($RANDOM%2))"
+		done
```

Comment:
There are currently 8 timing stress tests -- no clue why they weren't in the Jenkins job? That may be worth figuring out.

Reply:
@keithbostic it looks like 1-7 are used in `src/btree/bt_split.c` and 8 is used in `src/btree/bt_walk.c`. I found this in JIRA, might be related? [WT-4171: Enabling tree walk timing stress causes excessive slowdown](https://jira.mongodb.org/browse/WT-4171)